Sources/Noora/Utilities/TerminalText.swift

```diff
@@ -67,6 +74,12 @@ public struct TerminalText: Equatable, Hashable {
         }
         .joined()
     }
+
+    private func pathRelativeToWorkingDiretory(_ path: AbsolutePath) -> String {
+        guard let workingDirectoryPathString = ProcessInfo.processInfo.environment["PWD"],
```

Comment:
In Windows this will most likely result in showing the absolute path, which is fine for the time being. In the future we might want to iterate on this to have better support for Windows.